assets/stylesheets/bootstrap/_buttons.scss

```diff
@@ -96,8 +96,8 @@ a.btn {
 
 // Make a button look and behave like a link
 .btn-link {
-  font-weight: 400;
   color: $link-color;
+  border-color: transparent;
```

Comment:
Ensure `btn-link` always has a transparent border, even if it is also a `btn-danger`

Examples are 
- the cancel button on the action bar when item has been cut
![image](https://user-images.githubusercontent.com/5774647/94945252-53c51400-04d2-11eb-9bc3-7d917429c6e8.png)
- the set importance button when no importance has been set
![image](https://user-images.githubusercontent.com/5774647/94945308-63dcf380-04d2-11eb-9357-f9fee22ec192.png)